src/App.tsx

```diff
@@ -1,16 +1,23 @@
-import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+import './App.scss';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+  const handleAddMovie = (movie: Movie) => {
```

Comment:
Add empty space above to logical separate code
```suggestion
  
  const handleAddMovie = (movie: Movie) => {
```